Template.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <!--Put Title Here-->
+    <title>Front Page</title>
+    <style>
+
+        .content {
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            -moz-transform: translateX(-50%) translateY(-50%);
+            -webkit-transform: translateX(-50%) translateY(-50%);
+            transform: translateX(-50%) translateY(-50%);
+            width: 80;
+            margin: auto;
+        }
+
+        .pre-content {
+            background-color: #F9F9F9;
+            width: 100vw;
+            height: 100vh;
+        }
+
+        label {
+            font-weight: normal !important;
+        }
+
+        table {
+            border-collapse: collapse !important;
+        }
+
+        tr {
+            border: none !important;
+        }
+
+        td {
+            border: none !important;
+        }
+
+        .input {
+            height: 30px !important;
+        }
```

Comment:
Perhaps clean up unnecessary spacing here

Reply:
I like the spacing for clarity